airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlQueryUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+package io.airbyte.integrations.source.mssql;
+
+import static io.airbyte.cdk.integrations.source.relationaldb.RelationalDbQueryUtils.getFullyQualifiedTableNameWithQuoting;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.microsoft.sqlserver.jdbc.SQLServerException;
+import io.airbyte.cdk.db.jdbc.JdbcDatabase;
+import io.airbyte.cdk.db.jdbc.JdbcUtils;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteStream;
+import java.sql.SQLException;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class to define constants related to querying mssql
+ */
+public class MssqlQueryUtils {
+  private static final Logger LOGGER = LoggerFactory.getLogger(MssqlQueryUtils.class);
+  public static final String INDEX_QUERY = "EXEC sp_helpindex N'%s'";
+  public record Index(
+      @JsonProperty("index_name") String name,
+      @JsonProperty("index_description") String description,
+      @JsonProperty("index_keys") String keys
+  ) { }
+
+  public static void getIndexInfoForStreams(final JdbcDatabase database, final ConfiguredAirbyteCatalog catalog, final String quoteString) {
```

Comment:
can we rename this, seeing as it's not getting anything? The fact that it's only a logging function should be reflected in the name